internal/server/data/grant.go

```diff
@@ -151,65 +232,3 @@ func DeleteGrants(tx WriteTxn, opts DeleteGrantsOptions) error {
 	_, err := tx.Exec(query.String(), query.Args...)
 	return err
 }
-
-func ByOptionalPrivilege(s string) SelectorFunc {
-	return func(db *gorm.DB) *gorm.DB {
-		if s == "" {
-			return db
-		}
-
-		return db.Where("privilege = ?", s)
-	}
-}
-
-func GrantsInheritedBySubject(subjectID uid.PolymorphicID) SelectorFunc {
-	return func(db *gorm.DB) *gorm.DB {
-		switch {
-		case subjectID.IsIdentity():
-			userID, err := subjectID.ID()
-			if err != nil {
-				logging.Errorf("invalid subject id %q", subjectID)
-				return db.Where("1 = 0")
-			}
-			var groupIDs []uid.ID
-			err = db.Session(&gorm.Session{NewDB: true}).Raw("select distinct group_id from identities_groups where identity_id = ?", userID).Pluck("group_id", &groupIDs).Error
-			if err != nil {
-				logging.Errorf("GrantsInheritedByUser: %s", err)
-				_ = db.AddError(err)
-				return db.Where("1 = 0")
-			}
-
-			subjects := []string{subjectID.String()}
-			for _, groupID := range groupIDs {
-				subjects = append(subjects, uid.NewGroupPolymorphicID(groupID).String())
-			}
-			return db.Where("subject in (?)", subjects)
-		case subjectID.IsGroup():
-			return BySubject(subjectID)(db)
-		default:
-			panic("unhandled subject type")
-		}
-	}
-}
-
-func ByPrivilege(s string) SelectorFunc {
```

Comment:
reminder that any selector that wasn't prefixed with Optional was required, and not to change the functionality to optional